src/libsyncengine/update_detection/file_system_observer/localfilesystemobserverworker.cpp

```diff
@@ -180,16 +203,16 @@ void LocalFileSystemObserverWorker::changesDetected(const std::list<std::pair<st
                 if (!itemId.empty()) {
                     // Remove it from snapshot
                     _snapshot->removeItem(itemId);
-                    LOGW_SYNCPAL_DEBUG(_logger, L"Item removed from sync because it's hidden: "
-                                                        << Utility::formatSyncPath(absolutePath).c_str());
+                    LOGW_SYNCPAL_DEBUG(_logger,
+                                       L"Item removed from sync because it's hidden: " << Utility::formatSyncPath(absolutePath));
```

Comment:
```suggestion
                                       L"Item removed from sync because it is hidden: " << Utility::formatSyncPath(absolutePath));
```

Reply:
Why not...

Reply:
Done